sn_interface/src/network_knowledge/mod.rs

```diff
@@ -614,6 +614,14 @@ impl NetworkKnowledge {
         self.chain.has_key(key)
     }
 
+    /// Return weather current section chain has the provided key
```

Comment:
```suggestion
    /// Return a vec of all known keys
```

But also this doc doesn't seem t